scripts/ci/gitlab/pipeline/check.yml

```diff
@@ -68,7 +68,12 @@ check-try-runtime:
         echo "Found label runtimemigration. Running tests"
         export RUST_LOG=remote-ext=debug,runtime=debug
         echo "---------- Running try-runtime for ${NETWORK} ----------"
-        time cargo run --release --features=try-runtime try-runtime --chain=${NETWORK}-dev --execution=Wasm --no-spec-check-panic on-runtime-upgrade live --uri wss://${NETWORK}-try-runtime-node.parity-chains.parity.io:443
+        time cargo build --release -p "$NETWORK"-runtime
+        time cargo run --release --features try-runtime try-runtime \
+            --runtime ./target/release/wbuild/"$NETWORK"-runtime/target/wasm32-unknown-unknown/release/"$NETWORK"_runtime.wasm \
+            -lruntime=debug \
+            --chain=${NETWORK}-dev \
```

Comment:
Sorry that my release notes were not good enough to tell this: we should add a new `--checks` to the end of this as well.